crates/types/src/config/common.rs

```diff
@@ -478,14 +478,23 @@ impl Default for MetadataStoreClientOptions {
     schemars(title = "Tracing", description = "Options for tracing")
 )]
 pub struct TracingOptions {
-    /// # Tracing Endpoint
+    /// # Runtime Tracing Endpoint
     ///
-    /// Specify the tracing endpoint to send traces to.
+    /// Specify the tracing endpoint to send runtime traces to.
     /// Traces will be exported using [OTLP gRPC](https://opentelemetry.io/docs/specs/otlp/#otlpgrpc)
     /// through [opentelemetry_otlp](https://docs.rs/opentelemetry-otlp/0.12.0/opentelemetry_otlp/).
     ///
     /// To configure the sampling, please refer to the [opentelemetry autoconfigure docs](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#sampler).
```

Comment:
should we have these two be both settable by tracing_endpoint still? so that existing configs keep working

Reply:
Yup, I agree!